spec/acceptance/idp_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 feature 'IdpController' do
   scenario 'Login via default signup page' do
-    saml_request = make_saml_request('http://foo.example.com/saml/consume')
+    saml_request = custom_saml_request(overrides: { assertion_consumer_service_url: 'http://foo.example.com/saml/consume'})
```

Comment:
supernit: needs a space inside at the end
```suggestion
    saml_request = custom_saml_request(overrides: { assertion_consumer_service_url: 'http://foo.example.com/saml/consume' })
```

Reply:
good catch, updated in https://github.com/18F/saml_idp/pull/124/commits/8658a88a0e129e2a06f451b4e63e65c2dffa09ce